controller/include/pid.h

```diff
@@ -20,8 +20,10 @@ limitations under the License.
 #include "hal.h"
 #include "network_protocol.pb.h"
 
-inline constexpr AnalogPinId DPSENSOR_PIN = AnalogPinId::HAL_A0;
-inline constexpr PwmPinId BLOWERSPD_PIN = PwmPinId::PWM_3;
+inline constexpr AnalogPinId DPSENSOR_PIN = AnalogPinId::HAL_A5;
```

Comment:
Can this link to any documentation defining this to be the case? If not, I think it would make sense to create a wiki page describing the device interface and mentioning which pin is for what - in that case, feel free to just create a blank page with a TODO and mention it here, I'll fill it in.

Reply:
Agreed, but let me address this in a later PR if that's OK -- I'm consolidating all of this into HAL.  Then not only will all of these be in one place, but the *link* can also appear just once...